spec/mailers/previews/company_mailer_preview.rb

```diff
@@ -1,11 +1,11 @@
 class CompanyMailerPreview < ActionMailer::Preview
-  def decline_email
-    quote = Quote.last
-    CompanyMailer.decline_email(quote)
+  def decline_email_preview
+    # quote = Quote.last
+    CompanyMailer.decline_email(Quote.last)
```

Comment:
this is dangerous. There's no way to guarantee this was the last quote. You could pass the quote from the controller that calls this method.

```ruby
def decline_email_preview(quote)
  CompanyMailer.decline_email(quote)
end
```